### ISSUE TITLE
About policy loss

### ISSUE THREAD

**Issue by jzstudent** [shariqiqbal2810/maddpg-pytorch#20]
Could you tell me how to understand this "pol_loss += (curr_pol_out**2).mean() * 1e-3" in policy loss?

**Comment by shariqiqbal2810** [comment]
This term regularizes the policy output such that it doesn't get too large.
